akka-actor/src/main/java-jdk9/akka/util/Unsafe.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.util;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+/**
+ * INTERNAL API
+ */
+public final class Unsafe {
```

Comment:
Will add more comments here, but notice that this file can assume being JDK9

`akka-actor/src/main/java-jdk9/akka/util/Unsafe.java`